includes/embeds/class-amp-gallery-embed.php

```diff
@@ -113,9 +119,24 @@ public function shortcode( $attr ) {
 			);
 		}
 
-		return $this->render( array(
+		$args = array(
 			'images' => $urls,
-		) );
+		);
+		if ( ! empty( $atts['lightbox'] ) ) {
+			$args['lightbox'] = true;
+			$lightbox_tag     = AMP_HTML_Utils::build_tag(
+				'amp-image-lightbox',
+				array(
+					'id'                           => AMP_Base_Sanitizer::AMP_IMAGE_LIGHTBOX_ID,
```

Comment:
Why the static `'amp-image-lightbox-1'`? Should this not have some auto-increment number instead that is appended?

Reply:
Used `'amp-image-lightbox-1'` originally since thought that maybe there will be more than one, however, looks like using one should be enough, maybe we could just leave it as `'amp-image-lightbox'`. Thoughts?